includes/admin/class.llms.admin.builder.php

```diff
@@ -608,7 +608,7 @@ public static function output() {
 					echo self::get_template(
 						$template,
 						array(
-							'course_id' => $course_id,
+							'course_id' => $course_id, // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 						)
 					);
 				}
```

Comment:
I think this comment would need to be above the echo

Reply:
I agree. It reads correctly that way. For some reason the PCP was complaining unless the comment was on that line. I think we should refactor slightly so the code reads better and appeases the scanner.

Reply:
Added multi-line ignore to make the scanner happy. We could have probably one-lined the method as well I believe.